code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -195,6 +195,8 @@
 	//var/mob/living/carbon/human/H = user
 	var/row_num = 0
 	var/col_count = min(7,storage_slots) -1
+	if(storage_slots <= 0)
```

Comment:
This will be almost always true.
Don't you mean, `col_count`

Reply:
That's the point, it will almost be always true so that it defaults to showing 7 inventory slots if storage is 0
But col_count being set to 6 only if col_count ends up too small would be nicer